lib/process-node-definitions.js

```diff
@@ -9,6 +9,7 @@ var voidElementTags = [
 ];
 
 function ProcessNodeDefinitions() {
+  function preprocessDefaultNode(node, children, index) { }
```

Comment:
What's the purpose of this function? I can't see it being used.

Reply:
Its just there to add a default preprocessing instruction which does nothing at the moment. I added it just for consistency of having a default function similar to `processDefaultNode`. Its not used for now so I can remove it if you want.